docs/en/operations/system_tables.md

```diff
@@ -391,7 +391,7 @@ Columns:
 - `query` (String) — Query string.
 - `exception` (String) — Exception message.
 - `stack_trace` (String) — Stack trace (a list of methods called before the error occurred). An empty string, if the query is completed successfully.
-- `is_initial_query` (UInt8) — Kind of query. Possible values:
+- `is_initial_query` (UInt8) — Query type. Possible values:
```

Comment:
Either "query type" and "kind of query" are too generic, it just should explain what it means right in the first sentence.